### PR TITLE
Time in model training benchmark

### PR DESCRIPTION
**Pull request by karl-richter**
Added training time to the benchmarks + only hide the figures, but not the metrics.

### CODE REVIEW

**Comment by karl-richter** [comment]
@Kevin-Chen0 @ourownstory @noxan 
Can someone review and merge this pr asap pls? Need the time in the benchmarks to check that the Lightning speedup works.

File: .github/workflows/metrics.yml [hunk 1]
```diff
@@ -39,12 +39,12 @@ jobs:
       - name: Compare performance
         run: |
           echo "## Model Benchmark" >> report.md
-          echo "<details>\n<summary>Show benchmark results</summary>\n" >> report.md
           python tests/metrics/compareMetrics.py >> report.md
       - name: Publish report
         env:
           REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          echo "<details>\n<summary>Model training plots</summary>\n" >> report.md
```
Comment (noxan) [inline .github/workflows/metrics.yml:47]: I'd prefer to have an option to also collapse the regular metrics - no hard feelings and no blocker for sure 👍

**Review by noxan — APPROVED** [review]
Looks good to me, one minor comment on collapsible metrics - ready to merge from my side.

File: .github/workflows/metrics.yml (hunk 1 above)
Comment (karl-richter) [reply .github/workflows/metrics.yml:47]: Totally, agree, in the current setting there are too many metrics. Lets agree in our weekly meeting on the few key metrics that we always display and make all the others collabsable. I am just afraid that people do not check their metrics if its not visible that their pr reduces performance.

**Pull request merged**